launch-template.tf

```
@@ -90,10 +91,12 @@ resource "aws_launch_template" "default" {
   #     If any containers that you deploy to the node group use the Instance Metadata Service Version 2,
   #     then make sure to set the Metadata response hop limit to 2 in your launch template.
   metadata_options {
-    http_put_response_hop_limit = 2
     # Despite being documented as "Optional", `http_endpoint` is required when `http_put_response_hop_limit` is set.
     # We set it to the default setting of "enabled".
-    http_endpoint = "enabled"
+
+    http_endpoint               = var.launch_template_metadata_options_http_endpoint
+    http_put_response_hop_limit = var.launch_template_metadata_options_http_put_response_hop_limit
+    http_tokens                 = "required"
```

This needs to be configurable
```suggestion
    http_tokens                 = var.launch_template_metadata_options_http_tokens
```

---

@Nuru bridgecrew suggested this
https://docs.bridgecrew.io/docs/bc_aws_general_31

---

The Bridgecrew suggestion is the simplest possible solution. We need this to be configurable; we do not want to force everyone using our module to require tokens.